src/test/java/ibm/jceplus/junit/base/BaseTestRSAPSSInterop.java

```diff
@@ -482,7 +482,8 @@ protected void dotestSignature(byte[] content, String algorithmA, String algorit
             algParams.getParameterSpec(PSSParameterSpec.class);
             //System.out.println("parameters=" + algParams.toString());
         } else if (providerA.equals("SunRsaSign")) {
-            sig.setParameter(PSSParameterSpec.DEFAULT);
+            sig.setParameter(new PSSParameterSpec("SHA-1", "MGF1",
```

Comment:
The PSSParameterSpec.DEFAULT will new a PSSParameterSpec instance which has the below default values:
```
message digest  -- "SHA-1"
mask generation function (mgf) -- "MGF1"
parameters for mgf -- MGF1ParameterSpec.SHA1
SaltLength   -- 20
TrailerField -- 1
```
Why we need to call its constructor instead of using `PSSParameterSpec.DEFAULT` ?

Reply:
`PSSParameterSpec.DEFAULT` has been deprecated starting on Java20.